### PR TITLE
Align the table headers in all activity feed cards

### PR DESCRIPTION
**Pull request by ian-leggett**
## Description
The cards table content(headers) in the activity feed are all miss-aligned. As the structure slightly differs between a referral and interaction I have now set a minimum width for both referral and interactions so they align correctly.

## Before
Screenshot 2020-03-25 at 10 01 29

## After
Screenshot 2020-03-25 at 13 46 14

### CODE REVIEW

File: src/activity-feed/activities/card/CardDetailsList.jsx [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-const StyledUList = styled('ul')`
-  margin: 0;
-  padding-left: 0;
+const StyledListItem = styled(ListItem)`
+  ${MEDIA_QUERIES.TABLET} {
+    font-size: 16px;
+  }
 `
 
 export default class CardDetailsList extends React.PureComponent {
```
Comment (debitan) [inline src/activity-feed/activities/card/CardDetailsList.jsx]: OPTIONAL: this looks an easy one to convert to a functional component if you fancied it

File: src/activity-feed/activities/Referral.jsx [hunk 2]
```diff
@@ -50,6 +50,7 @@ export default class Referral extends React.PureComponent {
         />
 
         <CardTable
+          isReferral={true}
```
Comment (debitan) [inline src/activity-feed/activities/Referral.jsx]: is this possibly too specific? I know that this component as a whole is too specific for a shared component, but we could start here and rename this prop to something that reflects the change in style? Not sure exactly what that could be, maybe `isWideCard` or `noBottomMargin` card?

File: src/activity-feed/activities/card/CardDetailsList.jsx (hunk 1 above)
Comment (ian-leggett) [reply src/activity-feed/activities/card/CardDetailsList.jsx]: Agreed, its worth tidying this component up as we go.

File: src/activity-feed/activities/Referral.jsx (hunk 2 above)
Comment (ian-leggett) [reply src/activity-feed/activities/Referral.jsx]: I couldn't think of a better name to be fair so I used `isWideCard` I think this works thanks

Comment (mforner13) [reply src/activity-feed/activities/Referral.jsx]: @ian-leggett How would a user of the Card component know whether they should have isWideCard true or not? Could you name the prop after which bit of the referral card affects the alignment?

Comment (ian-leggett) [reply src/activity-feed/activities/Referral.jsx]: Good point, after our chat I have changed the prop to `isNotWrappedInDetails` 👍

**Pull request merged**